### DISCUSSION TITLE
Slip activity

### DISCUSSION THREAD

**Discussion by hchaieb1234** [mtex-toolbox/mtex#625] [Ask Anything]
Hello,

I have EBSD data and I want to study the activity of the different slip systems/twinning systems before deformation. Could you please tell me how to study the activation of the different systems and how to perform schmid factors maps for basal, pyramidal, prismatic systems..?

Thank you!

**Comment by lfgmorales** [comment]
Perhaps you could start here

https://mtex-toolbox.github.io/SchmidtFactor.html

Thanks

Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 9 Dec 2020, at 10:20, hchaieb1234 <notifications@github.com> wrote:

﻿

Hello,

I have EBSD data and I want to study the activity of the different slip systems/twinning systems before deformation. Could you please tell me how to study the activation of the different systems and how to perform schmid factors maps for basal, pyramidal, prismatic systems..?

Thank you!

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/625>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5INZZSY4C6U3SADJFWW3ST46NJANCNFSM4UTGFXEA>.

[ { "@context": "http://schema.org", "@type": "EmailMessage", "potentialAction": { "@type": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/625", "url": "https://github.com/mtex-toolbox/mtex/discussions/625", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { "@type": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Reply by hchaieb1234** [reply]
Thank you. Is there a way to identify twinning systems the same as basal/pyramidal? Is there an equivalent to:
sSBasal = slipSystem.basal(ebsd.CS);
